web/client/plugins/tutorial/selectors.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2017, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const selectors = {
```

Comment:
Remove this file, the selectors goes directly in the rawSteps array. Instead, create presets for home and map to use. 
So in your localConfig.json you will have something like` {name: "Tutorial", preset: "home"}` or ` {name: "Tutorial", preset: "map"}`